kubetest/gke.go

```diff
@@ -259,6 +261,17 @@ func (g *gkeDeployer) Up() error {
 			return err
 		}
 	}
+	// Create a custom subnet in that network if it was asked for.
```

Comment:
Seems like this custom subnet will never get cleaned up? If this custom subnet already exist, should we just use it?

Reply:
I think janitor will clean up custom subnet eventually - but check resource leaking might fail